core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -158,8 +158,22 @@ object ConsumerPerformance {
     val elapsedMs: Double = endMs - startMs
     val totalMBRead = (bytesRead * 1.0) / (1024 * 1024)
     val mbRead = ((bytesRead - lastBytesRead) * 1.0) / (1024 * 1024)
+    
```

Comment:
Thanks for the reply.

Here you changed the code from 
```
1000.0 * (mbRead / elapsedMs), messagesRead, ((messagesRead - lastMessagesRead) / elapsedMs) * 1000.0))
```
to 
```
val mbReadPerSec =
if ((endMs == startMs) && (mbRead == 0.0))
  0.0
else
1000.0 * (mbRead / elapsedMs)

val nMsgPerSec =
if ((endMs == startMs) && (messagesRead == lastMessagesRead))
  0.0
else
  ((messagesRead - lastMessagesRead) / elapsedMs) * 1000.0

```

The reason is because we want to handle the case where `endMs == startMs`. But since we will only call this method when `currentTimeMillis - lastReportTime >= config.reportingInterval` is true, then it is guaranteed that `endMs > startMs`, thus making the code change unnecessary, right?